src/Pages/Nodes.razor

```diff
@@ -204,6 +214,43 @@
             }
         }
     }
+    
+    private string _NodesState(Node node)
+    {
+        return node.IsNodeEnabled ? "Disable Node" : "Enable Node";
+    }
+    
+    private async Task ToggleNodeStatus(Node node)
+    {
+        try
+        {
+            node.IsNodeEnabled = !node.IsNodeEnabled;
+            if (!node.IsNodeEnabled)
+            {
+                await StopJob(node);
```

Comment:
this deletes the job, it should unschedule the job.
Disabling node: uneschedule job
Deleting node: delete job

Reply:
As we discuss before to delete the job.